src/goto-symex/goto_symex.h

```diff
@@ -80,6 +80,39 @@ class goto_symext
     const goto_functionst &goto_functions,
     const goto_programt &goto_program);
 
+  /// Symexes from the first instruction and the given state, terminating as
+  /// soon as the last instruction is reached.  This useful to explicitly symex
+  /// certain ranges of a program, e.g. in an incremental decision procedure.
+  /// \param state Symex state to start with.
+  /// \param goto_functions GOTO model to symex.
+  /// \param first Entry point in form of a first instruction.
+  /// \param last Final instruction, which itself will not be symexed anymore.
+  virtual void operator()(
+    statet &state,
+    const goto_functionst &goto_functions,
+    goto_programt::const_targett first,
+    goto_programt::const_targett last);
+
+  /// Initialise the symbolic execution and the given state with <code>pc</code>
+  /// as entry point.
+  /// \param state Symex state to initialise.
+  /// \param goto_functions GOTO model to symex.
+  /// \param pc Entry point in the form of a GOTO instruction.
+  /// \param end_of_function <code>END_FUNCTION</code> or final instruction to
```

Comment:
Rename if it doesn't have to be end-function. Also note this description is inconsistent with the above -- is this instruction included or not?